### PR TITLE
Add Biconomy to third party section

### PR DESCRIPTION
**Pull request by VGabriel45**
- Added information about the Biconomy SDK
- Included instructions for installing the SDK
- Provided an example on how to send a sponsored user operation

---

### Detailed summary
- Added Biconomy as a third-party library for integrating with ERC-4337
- Updated the documentation to include instructions for using Biconomy
- Added a new section for Biconomy with installation and usage instructions

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Review by jxom — CHANGES_REQUESTED** [review]
Thanks for the PR! Please add Biconomy to the end of the list as the page here is served on a first-in-first-serve basis (exception to sponsors).

File: site/pages/docs/third-party/account-abstraction.md [hunk 1]
```diff
@@ -1,12 +1,75 @@
 # Account Abstraction [Integrate Account Abstraction (ERC-4337) into viem.]
 
-While Account Abstraction is not built into the core `viem` library, you can use a third-party library like [permissionless.js](https://docs.pimlico.io/permissionless/reference) and [ZeroDev](https://docs.zerodev.app/) to integrate with ERC-4337.
+While Account Abstraction is not built into the core `viem` library, you can use a third-party library like [Biconomy](https://docs.biconomy.io/), [permissionless.js](https://docs.pimlico.io/permissionless/reference) and [ZeroDev](https://docs.zerodev.app/) to integrate with ERC-4337.
 
 **Libraries:**
 
+- [Biconomy](#biconomy)
 - [permissionless.js](#permissionless-js)
 - [ZeroDev](#zerodev)
```
[the remark this answers is not in the document]
Comment (VGabriel45) [reply site/pages/docs/third-party/account-abstraction.md]: Sure.

File: site/pages/docs/third-party/account-abstraction.md [hunk 2]
```diff
@@ -1,12 +1,75 @@
 # Account Abstraction [Integrate Account Abstraction (ERC-4337) into viem.]
 
-While Account Abstraction is not built into the core `viem` library, you can use a third-party library like [permissionless.js](https://docs.pimlico.io/permissionless/reference) and [ZeroDev](https://docs.zerodev.app/) to integrate with ERC-4337.
+While Account Abstraction is not built into the core `viem` library, you can use a third-party library like [Biconomy](https://docs.biconomy.io/), [permissionless.js](https://docs.pimlico.io/permissionless/reference) and [ZeroDev](https://docs.zerodev.app/) to integrate with ERC-4337.
```
[the remark this answers is not in the document]
Comment (VGabriel45) [reply site/pages/docs/third-party/account-abstraction.md]: Done

File: site/pages/docs/third-party/account-abstraction.md [hunk 3]
```diff
@@ -1,12 +1,75 @@
...
 
 **Libraries:**
 
+- [Biconomy](#biconomy)
 - [permissionless.js](#permissionless-js)
 - [ZeroDev](#zerodev)
 
+## Biconomy
```
[the remark this answers is not in the document]
Comment (VGabriel45) [reply site/pages/docs/third-party/account-abstraction.md]: done

**Review by jxom — APPROVED** [review]

**Pull request merged**